### DISCUSSION TITLE
MessageBox控件调用 ShowOverlayAsync方法出现异常

### DISCUSSION THREAD

**Discussion by 1184898069** [irihitech/Ursa.Avalonia#504] [Q&A]
如标题所示，在ViewModel中的方法中调用
    Result = await MessageBox.ShowOverlayAsync没有生效，没有模糊也没有弹窗显示；
但是在调用 
    Result = await MessageBox.ShowAsync时可以正常弹窗。
请问在使用ShowOverlayAsync需要有什么额外的设置吗，还是与其他控件有冲突呢

**Comment by rabbitism** [comment]
需要在界面上放置至少一个OverlayDialogHost（UrsaWindow内置一个所以不用放置）
